ui/mirage/handlers/clients.js

```diff
@@ -251,41 +249,41 @@ export default function (server) {
             // we don't currently use build_date, including for accuracy. it's only tracked in versions >= 1.11.0
             build_date: null,
             previous_version: null,
-            timestamp_installed: LICENSE_START.toISOString(),
+            timestamp_installed: '2023-07-02T00:00:00Z',
```

Comment:
Decided to hardcode these timestamps because using `add()` and `toISOstring()` caused the test to fail locally but pass on github, making these strings should resolve the problem